src/ppxgboost/PPTree.py

```diff
@@ -171,15 +154,17 @@ def eval(self, x):
 
         if x is None:
             raise RuntimeError("None in eval")
-        if self.feature_name not in x:
+
+        qd = x.query_dict
+        if self.feature_name not in qd:
             print('Feature name ' + self.feature_name + ' is not available in query')
```

Comment:
Change these to logging statements. `print()` (for debugging or logging) has no place in production code.